### PR TITLE
Added GitHubOwner to stack params

### PR DESCRIPTION
**Pull request by mjbuildit**

### CODE REVIEW

**Comment by aochsner** [comment]
does this require GITHUB_OWNER to always be specified?  I think this will make the default (if it's not specified) be blank `` not `buildit`

I haven't spent time to figure out how to optionally pass things to cfn via make.

Wonder if it's more user friendly just to add the owner to the front of the REPO?  `REPO=buildit/twig-api` for example

File: Makefile [hunk 1]
```diff
@@ -323,7 +323,7 @@ create-build: create-build-deps upload-build upload-lambdas
 			"ParameterKey=SsmNamespacePrefix,ParameterValue=/${OWNER}/${PROJECT}" \
 			"ParameterKey=SlackWebhook,ParameterValue=${SLACK_WEBHOOK}" \
 			"ParameterKey=Project,ParameterValue=${PROJECT}" \
-			"ParameterKey=Owner,ParameterValue=${OWNER}" \
```
Comment (aochsner) [inline Makefile:326]: unless i missed another change, Owner is still very much required

Comment (mjbuildit) [reply Makefile:326]: So as far as OWNER goes, that line was actually in there twice...

You make a good point with respect to overwriting the default template value with a blank string.  I'm inclined to just have the makefile also set the default if it doesn't already exist.

The problem with having the owner as part of the repo, IMO, is any scenario where the repo needs to be evaluated by itself, or something is being written to the local filesystem, or any combination of the two, where we end up having to parse the slash back out.  In particular, with what I'm working on right now, i.e., environment.declarations, it has to clone the repo to get the file, and then it has to locate the file in the repo directory, but the "owner" is not part of this, so the string has to be split.  Ultimately, I just think that if we combine owner and repo, then we lose information...which means that we have, conversely, increased entropy, and accelerated the heat death of the Universe.

Comment (aochsner) [reply Makefile:326]: doh...thanks wrt duplicate OWNER

yeah good point wrt owner in the repo...didn't think of that.  Makes sense

**Comment by aochsner** [comment]
LGTM

**Review by aochsner — APPROVED** [review]

**Pull request merged**